reload/ddsdk2016
# DDSDK-702

#### What does this PR do?
- Update our docker-compose setup a little bit so it supports orbstack-mac users better
- Cleans up the build tooling so it will watch for our changes
- Remove lost-pixel. We have moved on to argos so we need to implement that instead
- Updates our frontend action to run in correct location (our theme instead of root)
- Update `.platform.app.yaml` to build assets in theme and not root

#### How should this be manually tested?
1. Make sure that the PR-env looks like prod
2. Locally, check that it watches your changes to css and js etc.

Please note the `docker compose logs -f node`. It complains a lot about deprecated use of [Slash as division](https://sass-lang.com/documentation/breaking-changes/slash-div/) - I haven't done anything to that as it's mostly in node-bourbon package that is happening.

https://pr-495-rlzh7mq-55isd6w54kg6s.eu.platform.sh/
